lms/services/vitalsource/client.py

```diff
@@ -13,11 +16,43 @@ def __init__(self, lti_launch_key, lti_launch_secret):
         :type lti_launch_secret: str
         :raises ValueError: If credentials are invalid
         """
-        if not lti_launch_key or not lti_launch_secret:
-            raise ValueError("VitalSource LTI launch credentials are invalid")
+        if not all([lti_launch_key, lti_launch_secret, api_key]):
+            raise ValueError("VitalSource credentials are invalid")
```

Comment:
I guess they're "missing" rather than invalid:

```suggestion
        if not all([lti_launch_key, lti_launch_secret, api_key]):
            raise ValueError("VitalSource credentials are missing")
```

We normally wouldn't bother with this to be honest. I think the code just assumes that required settings are present and crashes if they aren't. It's usually obvious enough from the traceback what's wrong. But it's fine to do this as well